dynomitemanager-core/src/main/java/com/netflix/dynomitemanager/resources/DynomiteAdmin.java

```diff
@@ -163,10 +163,13 @@ public Response getClusterDescribe() {
 
 			for (AppsInstance ins : ii.getClusterInfo()) {
 				logger.debug("Adding node: " + ins.getInstanceId());
-				String node = "\"token\":" + "\"" + ins.getToken() + "\"," + "\"hostname\":" + "\"" + ins.getHostName()
-						+ "\"," + "\"rack\":" + "\"" + ins.getRack() + "\"," + "\"ip\":" + "\"" + ins.getHostIP()
-						+ "\"," + "\"zone\":" + "\"" + ins.getZone() + "\"," + "\"dc\":" + "\"" + ins.getDatacenter()
-						+ "\"";
+				String node = "\"token\":" + "\"" + ins.getToken() + "\","
+						+ "\"hostname\":" + "\"" + ins.getHostName() + "\","
+						+ "\"port\":" + "\"" + Integer.toString(ins.getDynomitePort()) + "\","
```

Comment:
what about the peer port. Do we need it?

Reply:
we can add that sure.